pom.xml

```diff
@@ -16,6 +16,9 @@
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<targetJdk>1.7</targetJdk>
+		<checkstyle.plugin.version>2.15</checkstyle.plugin.version>
+		<pmd.plugin.version>3.4</pmd.plugin.version>
+		<findbugs.version>3.0.0</findbugs.version>
```

Comment:
I used 3.0.0 because it was already used. We could consider a version bump of the tools. Are there any benefits? (Java 8 compatibility?)